xbmc/application/ApplicationPowerHandling.h

```diff
@@ -22,14 +24,17 @@ class IAddon;
 using AddonPtr = std::shared_ptr<IAddon>;
 } // namespace ADDON
 
+class CApplication;
 class CSetting;
 
 /*!
  * \brief Class handling application support for screensavers, dpms and shutdown timers.
  */
 
-class CApplicationPowerHandling
+class CApplicationPowerHandling : public IApplicationComponent
 {
+  friend class CApplication;
```

Comment:
I'm not sure what this is for?

Reply:
to give access to protected members which capp used to have access to through inheritance.